docs/cloud/azureai/manage-flows.md

```diff
@@ -102,6 +98,15 @@ flow = pf.flows.create_or_update(
 
 ::::
 
+On Azure portal, you can see the created flow in the flow list.
```

Comment:
To improve user guidance, it may be beneficial to instruct users to execute 'az login' as an initial step for acquiring credentials. 
At present, this information is not included in [prerequisites](https://microsoft.github.io/promptflow/cloud/azureai/quick-start.html#prerequisites) part, can only be found in the [run submit](https://microsoft.github.io/promptflow/cloud/azureai/quick-start.html#submit-a-run-to-workspace) part.  
The manage flow documentation only references the prerequisites, which could potentially result in users losing context.
![image](https://github.com/microsoft/promptflow/assets/32232238/95c12b59-983b-4d19-b2e3-84ede23648b0)